learn/writing-secure-ballerina-code.md

```diff
@@ -653,7 +653,7 @@ curl -k -v https://localhost:9091/hello
 Authentication failure
 ```
 
-Once a request is made with a valid, authentication information, but if the introspection endpoint does not respond with the "scope" attribute of the response JSON payload or respond with the "scope" attribute, which are not the expected scopes, an authorization failure will occur.
+Once a request is made with valid authentication information, but if the introspection endpoint does not respond with the "scope" attribute of the response JSON payload or respond with the "scope" attribute, which are not the expected scopes, an authorization failure will occur.
```

Comment:
```suggestion
If a request is made with valid authentication information, but the introspection endpoint does not respond with the "scope" attribute of the response JSON payload or respond with the "scope" attribute because these are not expected scopes, an authorization failure will occur.
```

Reply:
I'm still unclear about this part "does not respond with the "scope" attribute of the response JSON payload or respond with the "scope" attribute" - doesn't it say the same thing twice? Do they mean "...does or does not respond with the "scope" attribute of the response JSON payload"?

Reply:
Fixed.